cpp/build-support/run-test.sh

```diff
@@ -124,6 +119,42 @@ function run_test() {
   fi
 }
 
+function print_coredumps() {
+  # The script expects core files relative to the build directory with unique
+  # names per test executable because of the parallel running. So the corefile
+  # patterns must be set with prefix `core.{test-executable}*`:
+  #
+  # In case of macOS:
+  #   sudo sysctl -w kern.corefile=core.%N.%P
+  # On Linux:
+  #   sudo sysctl -w kernel.core_pattern=core.%e.%p
+  #
+  # and the ulimit must be increased:
+  #   sudo ulimit -c unlimited
```

Comment:
`sudo` is needless here.

Reply:
Umm. I couldn't push to `kszucs/arrow:coredump`.

@kszucs Could you remove the `sudo`?

Reply:
Done.